packages/coachmark/src/spectrum-config.js

```diff
@@ -13,60 +13,54 @@ governing permissions and limitations under the License.
 
 import { converterFor } from '../../../tasks/process-spectrum-utils.js';
 
-const converter = converterFor('spectrum-CoachMarkIndicator');
+const converter = converterFor('spectrum-CoachMark');
 
 /**
  * @type { import('../../../tasks/spectrum-css-converter').SpectrumCSSConverter }
  */
 const config = {
     conversions: [
         {
-            inPackage: '@spectrum-css/coachmark',
+            inPackage: '@spectrum-css/coachindicator',
             outPackage: 'coachmark',
-            fileName: 'coachmark',
-            excludeByComponents: [
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-image',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-content',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-footer',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-header',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-step',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-title',
-                },
-            ],
+            fileName: 'coach-indicator',
             components: [
-                converter.classToHost(),
-                converter.classToClass('spectrum-CoachMarkIndicator-ring'),
+                converter.classToHost('spectrum-CoachIndicator'),
+                converter.classToClass('spectrum-CoachIndicator-ring', 'ring'),
                 converter.classToAttribute(
-                    'spectrum-CoachMarkIndicator--quiet'
+                    'spectrum-CoachIndicator--quiet',
+                    'quiet'
                 ),
                 ...converter.enumerateAttributes(
                     [
-                        ['spectrum-CoachMarkIndicator--dark'],
-                        ['spectrum-CoachMarkIndicator--light'],
+                        ['spectrum-CoachIndicator--dark', 'black'],
+                        ['spectrum-CoachIndicator--light', 'white'],
```

Comment:
This appears to render `static="white"` in a black color... https://feature-coachmark-new--spectrum-web-components.netlify.app/storybook/?path=/story/coachindicator--static-white and visa versa https://feature-coachmark-new--spectrum-web-components.netlify.app/storybook/?path=/story/coachindicator--static-black

Reply:
Ping.